arcgis-ios-sdk-samples/Geometry/Geodesic operations/GeodesicOperationsViewController.swift

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class GeodesicOperationsViewController: UIViewController, AGSGeoViewTouchDelegate {
+
+    @IBOutlet private var mapView: AGSMapView!
+    
+    private let destinationGraphic: AGSGraphic
+    private let pathGraphic: AGSGraphic
+    
+    private let graphicsOverlay = AGSGraphicsOverlay()
+    private let measurementFormatter = MeasurementFormatter()
+    private let JFKAirportLocation = AGSPoint(x: -73.7781, y: 40.6413, spatialReference: AGSSpatialReference.wgs84())
+    
+    /// Add graphics representing origin, destination, and path to a graphics overlay.
+    required init?(coder aDecoder: NSCoder) {
+        // Create graphic symbols.
+        let locationMarker = AGSSimpleMarkerSymbol(style: .cross, color: .blue, size: 10)
```

Comment:
Looking at it on-device, I don't feel the locations are very clearly visible on the imagery basemap. Perhaps a different symbol color or shape would be better? Or perhaps a circle with an outline?

Other platforms are not much better, but perhaps at least we can match them with a circle?